aspnet/contribute/style-guide.rst

```diff
@@ -126,7 +126,7 @@ Numbered lists can start with a number, or they can be autonumbered by starting
 Source Code
 ^^^^^^^^^^^
 
-Source code is very commonly included in these articles. Images should never be used to display source code - instead use ``code-block`` or ``literalinclude``. You can refer to it using the ``code-block`` element, which must be declared precisely as shown, including spaces, blank lines, and indentation:
+Source code is very commonly included in these articles. Images should never be used to display source code. Use ``literalinclude`` generally or ``code-block`` for snippets not included in the sample project (``literalinclude`` is strongly preferred). You can refer to it using the ``code-block`` element, which must be declared precisely as shown, including spaces, blank lines, and indentation:
```

Comment:
Prefer ``literalinclude`` for most code samples. Reserve ``code-block`` for small snippets that are not included in the sample project.